defaults/main.yml

```diff
@@ -162,8 +160,7 @@ checkmk_server__webapi_url: '{{ checkmk_server__site_url + "/check_mk/webapi.py"
 # :ref:`checkmk_server__ref_omd_config` for more details.
 checkmk_server__omd_config: '{{
   checkmk_server__omd_config_email +
-  checkmk_server__omd_config_core +
-  (checkmk_server__omd_config_livestatus if checkmk_server__multisite_livestatus|d() else [])
+  checkmk_server__omd_config_core
 }}'
```

Comment:
I was just looking into this when I saw your PR. I would propose to change this list to a dict.